### DISCUSSION TITLE
What is query?

### DISCUSSION THREAD

**Discussion by kchen1023** [cmu-sei/GHOSTS#373] [Q&A]
![image](https://github.com/cmu-sei/GHOSTS/assets/131231802/664cbb86-ee21-4a41-a500-9216c81f6141)

I saw that we can search groups based on ID, but what should be the input for query? I tried "windows" but that doesn't seem to work...

**Comment by sei-dupdyke** [comment]
Searching by some querystring is not currently implemented, sorry.
